mo_sql_parsing/utils.py

```diff
@@ -276,10 +276,13 @@ def to_tuple_call(tokens):
     "similar_to": "similar_to",
     "like": "like",
     "rlike": "rlike",
+    "ilike": "ilike",
     "not like": "not_like",
     "not_like": "not_like",
     "not rlike": "not_rlike",
     "not_rlike": "not_rlike",
+    "not ilike": "not_ilike",
+    "not_ilike": "not_ilike",
```

Comment:
duplicate

Reply:
Yeah, actually I was confused by this. The same structure exists for `NOT LIKE` and `NOT RLIKE` just above, which is why I included it for `ILIKE` as well. I am not sure what function the key with underscore provides.

Reply:
oh! good catch!

Reply:
...no.  one has underscore the other a space.  Hmm.

Reply:
https://github.com/klahnakoski/mo-sql-parsing/commit/ef73fd7182ac83ba69afb6901ae339e166af7c3b